pkgdb2/lib/model/__init__.py

```diff
@@ -768,7 +768,7 @@ class PackageListing(BASE):
         sa.UniqueConstraint('package_id', 'collection_id'),
     )
 
-    package = relation("Package")
+    package = relation("Package", backref=backref('listing'),)
```

Comment:
This should probably be ``listings`` instead of ``listing`` since it is a one-to-many, not a one-to-one.